distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc

```diff
@@ -40,7 +40,7 @@
 |Attribute Overrides
 |attributeOverrides
 |String
-|Optional: Metacard attribute overrides (Key-Value pairs) that can be set on the content monitor. If an attribute is specified here, it will overwrite the metacard's attribute that was created from the content directory. The format should be 'key=value'.
+|Optional: Metacard attribute overrides (Key-Value pairs) that can be set on the content monitor. If an attribute is specified here, it will overwrite the metacard's attribute that was created from the content directory. The format should be 'key=value'. The maximum allowed size of an attribute override is 65,535 bytes.
```

Comment:
We'll also need this change in the same file on the `src/main/jdocs` path as well so it doesn't get lost when we transition to the templated docs.

Reply:
done